lib/xcpretty/formatters/rspec.rb

```diff
@@ -22,6 +23,10 @@ def format_pending_test(suite, test_case)
       yellow(PENDING)
     end
 
+    def format_measuring_test(suite, test_case, time)
```

Comment:
Unused method argument - `suite`. If it's necessary, use `_` or `_suite` as an argument name to indicate that it won't be used. You can also write as `format_measuring_test(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `test_case`. If it's necessary, use `_` or `_test_case` as an argument name to indicate that it won't be used. You can also write as `format_measuring_test(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `time`. If it's necessary, use `_` or `_time` as an argument name to indicate that it won't be used. You can also write as `format_measuring_test(*)` if you want the method to accept any arguments but don't care about them.